pyaedt/hfss3dlayout.py

```diff
@@ -1758,12 +1786,12 @@ def load_diff_pairs_from_file(self, filename):
         ----------
         >>> oModule.LoadDiffPairsFromFile
         """
-        if not os.path.isfile(filename):  # pragma: no cover
-            raise ValueError("{}: unable to find the specified file.".format(filename))
+        if not os.path.isfile(input_file):  # pragma: no cover
+            raise ValueError("{}: unable to find the specified file.".format(input_file))
```

Comment:
```suggestion
            raise ValueError("{}: Unable to find the specified file.".format(input_file))
```
Add concluding period. (I don't quite know where it goes.)